### PR TITLE
Feat/new defs

### PR DESCRIPTION
**Pull request by corymosiman12**
- add in initial haystack 3.9.10 with exploded point types
- package as cli tool

### CODE REVIEW

File: tasty/entities.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
+from copy import deepcopy
+from uuid import uuid4
+
+from rdflib import Namespace, RDF
+
+import tasty.graphs as tg
+import tasty.constants as tc
+
+
+class Instance:
```
Comment (aviveiros11) [inline tasty/entities.py:10]: I think generally all of the classes in this file need some documentation.  Items like bind aren't necessarily clear out of context .

File: tasty/generate_shapes.py [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+    # Here we translate the path into a BNode if necessary (i.e. for inverse pathing)
+    if each_path.get('path-type') is not None and each_path['path-type'] == 'inverse':
+        path_bn = BNode()
+        g.add((property_bn, SH.path, path_bn))
+        g.add((path_bn, SH.inversePath, namespaced_path))
+    elif each_path.get('path-type') is not None and each_path['path-type'] == 'direct':
+        g.add((property_bn, SH.path, namespaced_path))
+
+
+def add_things(g, ns_shape, shape, ontology):
```
Comment (aviveiros11) [inline tasty/generate_shapes.py:191]: "things" is unclear unless this is a concept in the semantic web world lol

**Review by aviveiros11 — COMMENTED** [review]
Just a couple of clarifying comments.  Should we also update the README to include a brief explanation of the `shape_json` fields?

File: tasty/generate_shapes.py (hunk 2 above)
Comment (corymosiman12) [reply tasty/generate_shapes.py:191]: Yeah, totally. will do this in separate stuff, tracking:
https://github.com/NREL/tasty/issues/8

File: tasty/entities.py (hunk 1 above)
Comment (corymosiman12) [reply tasty/entities.py:10]: Agreed. https://github.com/NREL/tasty/issues/7

**Pull request merged**